### ISSUE TITLE
Problem downloading files from dataverse

### ISSUE THREAD

**Issue by ennauata** [imatge-upc/slt_how2sign_wicv2023#9]
Hello,

Thanks for the great work. I'm trying to download the I3D keypoints and .tsv from the dataverse, but the download is really slow. 

I wonder if there's a faster option for downloading the data. 

Thanks.

**Comment by rabeya-akter** [comment]
Try downloading in collab .

from google.colab import drive
drive.mount("/content/gdrive")
!wget -c -r https://dataverse.csuc.cat/api/access/datafile/51543?gbrecs=true -O {"folder name/train.zip"}

this might fail 2/3 times. I could download the file in the third try.

Other files can be downloaded without any issues individually.

**Comment by ennauata** [comment]
Thank your for your response. I'm trying to download this one (https://dataverse.csuc.cat/dataset.xhtml?persistentId=doi%3A10.34810%2Fdata693). Would you know the right path for accessing through the API?

<img width="426" alt="image" src="https://github.com/imatge-upc/slt_how2sign_wicv2023/assets/719481/8cd60e2f-9885-4e3f-aca7-adb0cfff78e1">

**Comment by rabeya-akter** [comment]
> Thank your for your response. I'm trying to download this one (https://dataverse.csuc.cat/dataset.xhtml?persistentId=doi%3A10.34810%2Fdata693). Would you know the right path for accessing through the API?
> 
> <img alt="image" width="426" src="https://user-images.githubusercontent.com/719481/260071588-8cd60e2f-9885-4e3f-aca7-adb0cfff78e1.png">

No, I don't know about that. But the main problem happens while downloading the train.zip file. Other files can be downloaded individually quiet easily without any problem.

**Comment by ycmin95** [comment]
A fessible solution is using curl to [split the large file into chunks](https://www.maketecheasier.com/curl-command-guide/#split-merge-files), download them one by one, and merge them into the whole file. This solution works well for me:
```
curl --range 0-2000000000 -o part1 <download link for train.zip>
curl --range 2000000001-4000000000 -o part2 <download link for train.zip>
curl --range 4000000001-6000000000 -o part3 <download link for train.zip>
curl --range 6000000001- -o part4 <download link for train.zip>
cat part1 part2 part3 part4 > train.zip
```
The download results look like this:
![1694484430845](https://github.com/imatge-upc/slt_how2sign_wicv2023/assets/23148691/f343e48b-88d4-4ac3-a133-49f9b925245c)

**Comment by openhe-hub** [comment]
https://www.kaggle.com/datasets/rabeyaakter23/how2sign-i3d-features-mediapipe-features/data?select=i3d_features_how2sign  You can check this dataset from kaggle, I download it far faster than dataverse
